.gitlab-ci.yml

```diff
@@ -0,0 +1,196 @@
+---
+
+image: govcmsdev/gitlab-ci
```

Comment:
use quay.io/govcms/govcms-ci instead of this one - we're trying to get rid of it